server/cmwell-dc/src/pack/application.conf

```diff
@@ -30,7 +30,7 @@ akka {
 data-tools {
   akka {
     http.host-connection-pool {
-      max-connections = 20
```

Comment:
are you sure you want to change this number?

Reply:
Agree. My bad from earlier testing. Fixed